savant/deepstream/nvinfer/element_config.py

```diff
@@ -32,7 +33,9 @@
 from savant.remote_file import process_remote
 from savant.utils.logging import get_logger
 
-__all__ = ['nvinfer_element_configurator']
+__all__ = ['nvinfer_element_configurator', 'MERGED_CLASSES']
+
+MERGED_CLASSES = defaultdict(dict)
```

Comment:
@abramov-oleg maybe use per-model scopped mapping, not global? It may introduce collisions.

Reply:
Yes, I already use a two-level dict with the first level being tied to the element name. So, for example, multiple detector units based on the same detector model will not have any configuration collisions.